docs/custom-finders/getting-started.mdx

```diff
@@ -135,3 +168,19 @@ await $(ListTile).containing('Activated').$(#learnMore).tap();
 
 [patrol]: https://pub.dev/packages/patrol
 [create_finder_docs]: https://pub.dev/documentation/patrol/latest/patrol/createFinder.html
+
+### Falling back
+
+What's cool about Patrol is that it builds on top of `flutter_test` instead of
+replacing it. This means that you can freely mix Patrol's finders with finders
+from `flutter_test`, `PatrolTester` with `WidgetTester`, and so on.
+
+Here's how you can access the default `WidgetTester`:
+
+```dart
+patrolTest('adds comment', (PatrolTester $) async {
+  final WidgetTester tester = $.tester;
+
+  await tester(#commentTextField).enterText('Very nice!');
+}
```

Comment:
missing `);`

Reply:
thanks.

actually I must've phased out while writing this chunk because what I wrote doesn't make any sense.

thanks again